.github/workflows/ci.yml

```
@@ -218,12 +219,8 @@ jobs:
       # hwloc (opt out)
       - name: Install hwloc
         if: matrix.config.hwloc != 'no'
-        run: |
-          apt-get update
-          apt-get install -y libhwloc-dev pkg-config
-          which pkg-config
-          echo "hwlock version:" $(pkg-config --modversion hwloc)
-          dpkg -L libhwloc-dev
+        shell: bash
+        run: "${APCI_ALPAKA_ROOT}/script/ci/install/tbb.sh"
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:1: Double quote to prevent globbing and word splitting

(shellcheck)